src/atoms/Heading/Heading.native.js

```diff
@@ -1,85 +1,77 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components/native';
-import { Text as NativeText } from 'react-native';
-import { getColorKeys, getColor } from '../../_helpers/theme';
+import { useTheme } from 'styled-components';
```

Comment:
Just commit this directly here

```suggestion
import { useTheme } from 'styled-components/native';
```